examples/simple-source/src/main.rs

```diff
@@ -52,7 +54,8 @@ pub(crate) mod simple_source {
                 self.read_idx
                     .store(self.read_idx.load(Ordering::Relaxed) + 1, Ordering::Relaxed);
                 let offset = self.read_idx.load(Ordering::Relaxed);
-
+                headers.insert(String::from("x-txn-id"), String::from(Uuid::new_v4()));
```

Comment:
where are you defining headers? does this code even compile?

Reply:
sorry missed in copy pasting :( ,fixed it now